devel/external/test_photon_vs_fft/cosmos_sersics_sample_N300.asc

```diff
@@ -0,0 +1,301 @@
+# ident  n_sersic  half_light_radius [arcsec] |g|
```

Comment:
Why did you replace the symlink with a copy of the file (here and several more below in these devel/external directories)?  Please revert.  No need to clutter the repo with multiple copies of these files.  The symlinks work fine.